nucypher/cli/actions.py

```diff
@@ -179,11 +171,17 @@ def determine_external_ip_address(emitter, force: bool = False) -> str:
 
 def destroy_configuration(emitter, character_config, force: bool = False) -> None:
     if not force:
+        try:
+            database = character_config.db_filepath
+        except AttributeError:
+            database = "No database found"
```

Comment:
:question:

Reply:
So destroy works for other characters other than Ursula who don't have a database